docs/indepth/installation.rst

```diff
@@ -215,7 +215,7 @@ Using Amazon Web Services
 
    Kernels older than ``3.16.4`` have a bug that affects Flocker's use of ZFS.
```

Comment:
Expand this to identify that kernel must be after 3.16.4 and before 3.18.  Perhaps more like the Digital Ocean text?